library/modules/Screen.cpp

```diff
@@ -1081,7 +1081,7 @@ using df::identity_traits;
 #define CUR_STRUCT dfhack_viewscreen
 static const struct_field_info dfhack_viewscreen_fields[] = {
     { METHOD(OBJ_METHOD, is_lua_screen), 0, 0 },
-    { METHOD(OBJ_METHOD, getFocusString), 0, 0 },
+    { METHOD(OBJ_METHOD, getFocusStrings), 0, 0 },
```

Comment:
I believe this is the (singular) focus string provided by the DFHack viewscreen. it shouldn't be changed to the plural.